packages/autocomplete/src/elements/Autocomplete.example.md

```diff
@@ -0,0 +1,89 @@
+### Basic Example
+
+```jsx
+const items = ['Option 1', 'Option 2', 'Option 3'];
+const options = [];
+
+for (let x = 0; x < items.length; x++) {
+  options.push({
+    value: `value-${x}`,
+    label: items[x]
+  });
+}
+
+const optionsMap = options.reduce((dictionary, option) => {
+  dictionary[option.value] = option.label;
+  return dictionary;
+}, {});
+
+initialState = {
+  selectedValue: options[0].value
+};
+
+<Autocomplete
+  label="Basic Autocomplete"
+  placeholder={optionsMap[state.selectedValue]}
```

Comment:
replacing the value with a placeholder on menu expansion feels weird to me. It's not consistent with a true `<select>` dropdown and it's not consistent across browsers (placeholder disappears in IE). The value should remain intact and be replaced on keyboard entry. The placeholder should be whatever is displayed if a value is cleared.